sdk.zig

```diff
@@ -16,7 +16,9 @@ pub fn Sdk(comptime deps: anytype) type {
             _module = b.createModule(.{
                 .source_file = .{ .path = sdkPath("/src/main.zig") },
                 .dependencies = &.{
-                    .{ .name = "sysjs", .module = deps.sysjs.module(b) },
+                    .{ .name = "sysjs", .module = b.createModule(.{
+                        .source_file = .{ .path = "libs/mach-sysjs/src/main.zig" },
+                    }) },
```

Comment:
this is going to be problematic because sysjs module will not be deduplicated; that's the whole point of `deps.sysjs.module(b)`.

But this will go away when we depend on sysjs using the Zig package manager, so..